### PR TITLE
#611:Eliminate variable shadowing in local.c

### PR DESCRIPTION
**Pull request by msporcich**
Resolves build problems when compiling with -Werror=shadow.

Signed-off-by: [Marc Sporcich] <marc@epiqsolutions.com>

### CODE REVIEW

File: local.c [hunk 1]
```diff
@@ -1198,7 +1197,7 @@ static char * get_short_attr_name(struct iio_channel *chn, const char *attr)
 		ptr += len + 1;
 
 	if (chn->name) {
-		size_t len = strlen(chn->name);
+		len = strlen(chn->name);
```
Comment (commodo) [inline local.c:1200]: the commit also needs a Signed-off-by tag;

Comment (commodo) [reply local.c:1200]: and preferably the same description as in the PR;
Github PRs don't track too well in git history; so the information in the PR description is sightly more useful in the git history to be able to view it with `git log` on the command line

Comment (msporcich) [reply local.c:1200]: Ah yes, understood.

**Review by commodo — APPROVED** [review]

**Comment by commodo** [comment]
Thanks :)

**Pull request merged**